spec/jobs/browse_everything_ingest_job_spec.rb

```diff
@@ -0,0 +1,4 @@
+require 'rails_helper'
+
+RSpec.describe BrowseEverythingIngestJob, type: :job do
```

Comment:
Looks like BrowseEverythingIngestJob is covered by the ScannedResourcesController test, so maybe we should remove this file if we don't have any unit tests for it?